### DISCUSSION TITLE
Is Elk.zone down?

### DISCUSSION THREAD

**Discussion by Philomorph** [elk-zone/elk#1844] [Q&A]
I'm unable to login to either of my instances today through Elk.zone - tabletop.social or fosstodon.org. They both work straight from a web browser. A friend is having the same problem, so I know it's not just me.

Suggestions?

**Comment by patak-cat** [comment]
I'm also having issues with fosstodon.org now. I'll forward this to https://chat.elk.zone to see what other servers are affected.

**Reply by patak-cat** [reply]
@Philomorph deleted your last message in case the url you shared contained a token. We're seeing the same as you 👍🏼

**Reply by patak-cat** [reply]
@Philomorph please try again, it is resolved for me now 👍🏼

**Reply by Philomorph** [reply]
Yep, fosstodon is back online. But I still can't get on tabletop.social. Still gets "ERROR: TypeError: Failed to fetch"

**Reply by patak-cat** [reply]
I cleared the instance in elk.zone. Would you try again for it @Philomorph?

**Reply by Philomorph** [reply]
It asked me to reauthorize, but then still returns the same error afterward.

**Reply by patak-cat** [reply]
Did this instance work before? I think the issue may be different here, is it using a standard Mastodon server?
